job-portal/routes/auth.js

```diff
@@ -13,6 +13,20 @@ var crypto = require('crypto')
 var nodemailer = require('nodemailer')
 var Token = require('../models/Token')
 
+function passValidation(pass) {
+  counter = 0
```

Comment:
Umm you'll have to do it the Js way, 
```suggestion
  var counter = 0
```